lib/create/issue.js

```diff
@@ -27,18 +30,23 @@ async function open(req, res) {
 async function submit(req, res) {
   const { slackUser } = res.locals;
 
-  const { title, body } = req.body.submission;
+  const { title, body, label } = req.body.submission;
   const { repository } = req.body.submission;
 
   await slackUser.reload({ include: [GitHubUser] });
+
   const { data } = await slackUser.GitHubUser.client.repos.getById({ id: repository });
   const owner = data.owner.login;
   const repo = data.name;
+  const { data: list } = await slackUser.GitHubUser.client.issues.getLabels({ owner, repo });
```

Comment:
Can we avoid this HTTP request by passing the label name instead of the id on this line: https://github.com/integrations/slack/pull/626/files#diff-56aea2064d1a529dddab5a8b3e762507R41

I'd like to avoid additional HTTP requests as part of the dialog submission because we're on the clock 🕐 
Sorry for only bringing this up now.